### PR TITLE
New costs

### PR DESCRIPTION
**Pull request by arvidn**
this depends on https://github.com/Chia-Network/clvm/pull/14

### CODE REVIEW

File: clvm/more_ops.py [hunk 1]
```diff
@@ -156,7 +201,7 @@ def op_strlen(args):
     if a0.pair:
         raise EvalError("len on list", a0)
     size = len(a0.as_atom())
-    cost = size
+    cost = STRLEN_BASE_COST + size / STRLEN_COST_PER_BYTE_DIVIDER
```
Comment (richardkiss) [inline clvm/more_ops.py:204]: Do you mean `//` here?

Comment (arvidn) [reply clvm/more_ops.py:204]: ah, yes, I did

**Comment by richardkiss** [comment]
The divisions concern me a little bit, as I worry they will overwhelm the operations they are calculating the cost of. I do notice that they all tend to be powers of 2... what do you think about changing these to `>>`, which we know will run quickly.

**Comment by arvidn** [comment]
is there any doubt that `// 64` is not identical to `>> 6`? I think the former is much more legible

**Comment by richardkiss** [comment]
There's no doubt, I'm just concerned it's slower. Probably C or rust would optimize it, but I'm not sure that python does (since it has no assurances that what we humans consider a constant doesn't actually change).

**Comment by arvidn** [comment]
in C and C++ I would say `/` is preferred over `>>` because there are certain edge-cases where `>>` behaves unexpected, especially on signed variables.

**Comment by arvidn** [comment]
I rebased and fixed the conflicts with the `args-**` changes. I also fixed the floating-point division -> integer division. I applied that as a separate commit to ease review, but I think they should be squashed before landing.

**Comment by richardkiss** [comment]
```(venv) kiss@oasis develop % python -m timeit -s "T = 10004843716274856703848" "T >> 6"
5000000 loops, best of 5: 59.1 nsec per loop
(venv) kiss@oasis develop % python -m timeit -s "T = 10004843716274856703848" "T // 64"
5000000 loops, best of 5: 87.1 nsec per loop
(venv) kiss@oasis develop % python -m timeit -s "T = 100" "T // 64" 
10000000 loops, best of 5: 34.6 nsec per loop
(venv) kiss@oasis develop % python -m timeit -s "T = 100" "T >> 6" 
5000000 loops, best of 5: 59.1 nsec per loop
```

Our values on the left are likely to be closer to 100 than 10004843716274856703848, so turns out that `//` is within order of magnitude even in the worst case. My intuitions were off here.

**Pull request merged**